testing/CMakeLists.txt

```diff
@@ -55,6 +55,9 @@ if (CDAT_BUILD_MATPLOTLIB)
   add_subdirectory(matplotlib)
 endif()
 
+# Test ipython
+add_test(run_ipython ${CMAKE_INSTALL_PREFIX}/bin/ipython -V)
```

Comment:
@durack1 it should work, if the command is not found, I believe shell will set the error flag and ctest would know that test has failed. I can do local testing to make sure this works as expected but my initial reaction was that it should and that's why I approved the branch.